x-pack/plugins/actions/server/lib/task_runner_factory.ts

```diff
@@ -198,9 +199,11 @@ export class TaskRunnerFactory {
           logger.error(`Action '${actionId}' failed: ${executorResult.message}`);
           // Task manager error handler only kicks in when an error thrown (at this time)
           // So what we have to do is throw when the return status is `error`.
+
           throw throwRetryableError(
             new Error(executorResult.message),
-            executorResult.retry as boolean | Date
+            executorResult.retry as boolean | Date,
+            TaskErrorSource.USER
```

Comment:
Action executer either tackles the errors and return success or throw.
Only the validation error comes here.